roles/kubernetes-apps/ingress_controller/ingress_nginx/templates/ds-ingress-nginx-controller.yml.j2

```diff
@@ -46,6 +46,9 @@ spec:
             - --tcp-services-configmap=$(POD_NAMESPACE)/tcp-services
             - --udp-services-configmap=$(POD_NAMESPACE)/udp-services
             - --annotations-prefix=nginx.ingress.kubernetes.io
+{% if ingress_nginx_host_network %}
```

Comment:
I think this will leave a whitespace in the config causing it to fail

Reply:
Ansible v2.7 (and v2.6) runs Jinja2 with `trim_blocks` enabled and `lstrip_blocks` disabled by default.
https://docs.ansible.com/ansible/2.7/modules/template_module.html

This template is evaluated as expected when `ingress_nginx_host_network = true`.
```
$ ansible --version | head -n1
ansible 2.7.6
$ cd roles/kubernetes-apps/ingress_controller/ingress_nginx
$ ansible all -i "localhost," -c local -m template -a "src=templates/ds-ingress-nginx-controller.yml.j2 dest=/tmp/ds-ingress-nginx-controller.yml" --extra-vars=@defaults/main.yml --extra-vars=@../../../download/defaults/main.yml --extra-vars='{"kube_version": "1.13.2", "k8s_image_pull_policy": "IfNotPresent", "ingress_nginx_host_network": false}' >/dev/null
$ ansible all -i "localhost," -c local -m template -a "src=templates/ds-ingress-nginx-controller.yml.j2 dest=/tmp/ds-ingress-nginx-controller.yml" --extra-vars=@defaults/main.yml --extra-vars=@../../../download/defaults/main.yml --extra-vars='{"kube_version": "1.13.2", "k8s_image_pull_policy": "IfNotPresent", "ingress_nginx_host_network": true}' --check --diff
--- before: /tmp/ds-ingress-nginx-controller.yml
+++ after: /Users/youngchul/.ansible/tmp/ansible-local-703967JKWr3/tmpGIClvJ/ds-ingress-nginx-controller.yml.j2
@@ -22,6 +22,7 @@
         prometheus.io/scrape: "true"
     spec:
       serviceAccountName: ingress-nginx
+      hostNetwork: true
       nodeSelector:
         node-role.kubernetes.io/node: ''
       containers:
@@ -34,6 +35,7 @@
             - --tcp-services-configmap=$(POD_NAMESPACE)/tcp-services
             - --udp-services-configmap=$(POD_NAMESPACE)/udp-services
             - --annotations-prefix=nginx.ingress.kubernetes.io
+            - --report-node-internal-ip-address
           securityContext:
             capabilities:
                 drop:

localhost | CHANGED => {
    "changed": true
}
```